src/sentry/uptime/consumers/results_consumer.py

```diff
@@ -66,6 +67,8 @@
 # The TTL of the redis key used to track consecutive statuses
 ACTIVE_THRESHOLD_REDIS_TTL = timedelta(minutes=60)
 SNUBA_UPTIME_RESULTS_CODEC: Codec[SnubaUptimeResult] = get_topic_codec(Topic.SNUBA_UPTIME_RESULTS)
+# We want to limit cardinality for provider tags. This controls how many tags we should include
+PROVIDER_NAMES_TO_INCLUDE_AS_TAGS = 30
```

Comment:
```suggestion
TOTAL_PROVIDERS_TO_INCLUDE_AS_TAGS = 30
```

maybe this is more clear